### PR TITLE
Adding Quick Responses APIs

### PR DESCRIPTION
**Pull request by hngan**
Introducing two new APIs to support the new quick responses feature

### CODE REVIEW

**Review by vpogudina — APPROVED** [review]

File: src/event.js [hunk 1]
```diff
@@ -49,7 +49,15 @@
     'authorize_success',
     'authorize_retries_exhausted',
     'cti_authorize_retries_exhausted',
-    'click_stream_data'
```
Comment (seiyako) [inline src/event.js:52]: Can you remove non-QR events?

File: src/index.d.ts [hunk 2]
```diff
@@ -253,7 +253,11 @@ declare namespace connect {
     AUTHORIZE_SUCCESS = 'authorize_success',
     AUTHORIZE_RETRIES_EXHAUSTED = 'authorize_retries_exhausted',
     CTI_AUTHORIZE_RETRIES_EXHAUSTED = 'cti_authorize_retries_exhausted',
-    CLICK_STREAM_DATA = 'click_stream_data'
```
Comment (seiyako) [inline src/index.d.ts:256]: Same here, remove non-QR changes

File: src/mediaControllers/chat.js [hunk 3]
```diff
@@ -23,37 +23,39 @@
     var logComponent = connect.LogComponent.CHAT;
 
     var createMediaInstance = function () {
-      publishTelemetryEvent("Chat media controller init", mediaInfo.contactId);
```
Comment (seiyako) [inline src/mediaControllers/chat.js:26]: looks like linting applied to these files, but are these synced with the internal changes?

File: src/event.js (hunk 1 above)
Comment (hngan) [reply src/event.js:52]: removed

File: src/index.d.ts (hunk 2 above)
Comment (hngan) [reply src/index.d.ts:256]: removed

**Review by seiyako — APPROVED** [review]

**Pull request merged**